src/Compilers/VisualBasic/Portable/Binding/BinderFactory.vb

```diff
@@ -387,8 +387,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                  SyntaxKind.GetAccessorStatement,
                                  SyntaxKind.AddHandlerAccessorStatement,
                                  SyntaxKind.RemoveHandlerAccessorStatement,
-                                 SyntaxKind.RaiseEventAccessorStatement
-                                ' Default values are not valid (and not bound) for lambda parameters or property accessors
+                                 SyntaxKind.RaiseEventAccessorStatement,
+                                 SyntaxKind.OperatorStatement
```

Comment:
@AlekseyTs I'm not sure I'm following. The fix here is to change `CreateBinderForNodeAndUsage()` to return `Nothing` if the caller asks for a `NodeUsage.ParameterDefaultValue` with a `SyntaxKind.OperatorStatement`. Callers of `GetBinderForNodeAndUsage()` should expect and know how to handle return values of `Nothing`. Why should the upper level `VisitParameter()` check for specific node types if the underlying `BinderFactory` type is designed to do so? Am I missing something?